Extensions/Settings/Dnn.PersonaBar.SiteSettings/SiteSettings.Web/src/components/languageSettings/index.jsx

```diff
@@ -76,15 +68,6 @@ class LanguageSettingsPanelBody extends Component {
         }));
     }
 
-    componentDidUpdate(props) {

```

Comment:
Same here

Reply:
this method is actually useless. It has been removed and component has been refactored accordingly.